examples/tests/valid/website_with_api.w

```diff
@@ -9,9 +9,9 @@ let usersTable = new cloud.Table(
   name: "users-table",
   primaryKey: "id",
   columns: {
-    id: cloud.ColumnType.STRING,
-    name: cloud.ColumnType.STRING,
-    age: cloud.ColumnType.NUMBER,
+    id => cloud.ColumnType.STRING,
```

Comment:
Are we also allow keys of maps to be symbolic (`id` instead of `"id"`). I think we should only allow this for json.

```suggestion
    "id" => cloud.ColumnType.STRING,
```

Reply:
hmm, it would be an easy change to make but why would we not allow symbolic keys in maps?

Reply:
@hasanaburayyan - our maps are Map<str, T> (map from string to T), so let's enforce that map keys are strings if that's easy. If not, we can save that for a future PR

Reply:
I updated the grammar to only accept keys that are strings